perllib/FYR/Queue.pm

```diff
@@ -1135,6 +1140,85 @@ sub email_template_params ($%) {
     return \%params;
 }
 
+sub build_html_email {
+    my ($template, $msg, $settings) = @_;
+
+    my $html_settings = FYR::EmailSettings::get_settings;
+    foreach my $setting (keys %$settings) {
+        $html_settings->{$setting} = $settings->{$setting};
+    }
+
+    # Convert the plain text message into html so it displays
+    # more or less correctly in HTML emails. This splits the
+    # address portion and the message portion as they have
+    # different formatting requirements
+    if ($html_settings->{email_text}) {
+        my $msg = $html_settings->{email_text};
+        my ($address, $body) = split 'Email:', $msg;
+        $address =~ s%\n%<br/>%gs;
+        $body =~ s%\n\n%</p>\n<p>%gs;
+        $html_settings->{email_text} = "$address</p><p>Email:$body";
```

Comment:
Space after Email:?
And we chatted about if a `<p align="right">` might work for the address, no biggie.